src/goto-symex/goto_symex_state.cpp

```diff
@@ -306,6 +188,41 @@ static void assert_l2_renaming(const exprt &expr)
   #endif
 }
 
+class goto_symex_is_constantt : public is_constantt
+{
+protected:
+  bool is_constant(const exprt &expr) const override
+  {
+    if(expr.id()==ID_mult)
+    {
+      // propagate stuff with sizeof in it
```

Comment:
Why is this only the case for multiply?

(Again, I realise that this is not new code.)

Reply:
#3095 for further discussion.